lldb/examples/python/crashlog.py

```diff
@@ -1794,8 +1802,35 @@ def SymbolicateCrashLogs(debugger, command_args, result, is_command):
         result.SetError(str(e))
         return
 
+    # To avoid breaking existing users, we should keep supporting legacy flags
+    # even if we don't use them / advertise them anymore.
+    if not options.mode:
+        if options.batch:
+            options.mode = CrashLogLoadingMode.batch
+        else:
+            options.mode = CrashLogLoadingMode.interactive
```

Comment:
Can you detect both `options.batch` and `options.interactive` being specified at the same time? I assume whichever is listed first will get selected... but I would assume we'd want to add something that prevents you from choosing both.

Reply:
It's illegal to have both `batch` & `interactive` and `argparse` will prevent you from doing anything.